template/template.go

```diff
@@ -141,6 +141,7 @@ type Provisioner struct {
 	Config      map[string]interface{} `json:"config,omitempty"`
 	Override    map[string]interface{} `json:"override,omitempty"`
 	PauseBefore time.Duration          `mapstructure:"pause_before" json:"pause_before,omitempty"`
+	Retry       int                    `mapstructure:"retry" json:"retry,omitempty"`
```

Comment:
I would may be make of this MaxRetries or RetryLimit as it sounds clearer to me ? What do you think ?
```suggestion
	MaxRetries   int                    `mapstructure:"max_retries" json:"max_retries,omitempty"`
```

This could even be `on_error_max_retries` but may be that's too verbose 🤔

Reply:
I like max_retries.